sdk/scheduler/src/main/java/com/mesosphere/sdk/api/EndpointUtils.java

```diff
@@ -58,15 +58,26 @@ public static String toEndpoint(String hostname, int port) {
     }
 
     /**
-     * Returns the correct Mesos DNS endpoint for the provided task and port running within the provided service.
+     * Returns the correct DNS domain for tasks within the service.
      */
-    public static String toAutoIpEndpoint(String serviceName, String taskName, int port) {
+    public static String toAutoIpDomain(String serviceName) {
         // Unlike with VIPs and mesos-dns hostnames, dots are converted to dashes with autoip hostnames. See DCOS-16086.
-        String hostname = String.format("%s.%s.%s",
-                removeSlashes(replaceDotsWithDashes(taskName)),
-                removeSlashes(replaceDotsWithDashes(serviceName)),
-                AUTOIP_HOST_TLD);
-        return toEndpoint(hostname, port);
+        return String.format("%s.%s", removeSlashes(replaceDotsWithDashes(serviceName)), AUTOIP_HOST_TLD);
```

Comment:
There isn't, removed the `EndpointUtils` versions in favor of the `Constants` versions